packages/babel-plugin-extract-messages/test/fixtures/js-message-descriptor.js

```diff
@@ -1,11 +1,7 @@
-/*i18n*/'Message'
-
 const msg = /*i18n*/{id: 'Message'}
 
 const withDescription = /*i18n*/{id: 'Description', comment: "description"}
 
-const withMultilineComment = /*i18n*/{id: 'Multiline', comment: "this is " + "2 lines long"}
-
```

Comment:
Created a specific suite, and also tested negative cases where variable passed as value.